src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationModel.swift

```diff
@@ -0,0 +1,107 @@
+			]
+		)
+	])
+
+	static let contactModel = DynamicTableViewModel([
+		.section(
+			header: .image(UIImage(named: "Illu_Appinfo_Kontakt"),
+						   accessibilityLabel: AppStrings.AppInformation.contactImageDescription,
+						   accessibilityIdentifier: AccessibilityIdentifiers.AppInformation.contactImageDescription,
+						   height: 230),
+			cells: [
+				.title2(text: AppStrings.AppInformation.contactTitle,
+						accessibilityIdentifier: AccessibilityIdentifiers.AppInformation.contactTitle),
+				.body(text: AppStrings.AppInformation.contactDescription,
+					  accessibilityIdentifier: AccessibilityIdentifiers.AppInformation.contactDescription),
+				.headline(text: AppStrings.AppInformation.contactHotlineTitle,
+						  accessibilityIdentifier: AccessibilityIdentifiers.AppInformation.contactHotlineTitle),
+				.phone(text: AppStrings.AppInformation.contactHotlineText, number: AppStrings.AppInformation.contactHotlineNumber,
+					   accessibilityIdentifier: AccessibilityIdentifiers.AppInformation.contactHotlineText),
+				.footnote(text: AppStrings.AppInformation.contactHotlineDescription,
+						  accessibilityIdentifier: AccessibilityIdentifiers.AppInformation.contactHotlineDescription),
+				.footnote(text: AppStrings.AppInformation.contactHotlineTerms,
+						  accessibilityIdentifier: AccessibilityIdentifiers.AppInformation.contactHotlineTerms)
+			]
+		)
+	])
+
+
+	static let privacyModel = DynamicTableViewModel([
+		.section(
+			header: .image(
+				UIImage(named: "Illu_Appinfo_Datenschutz"),
+				accessibilityLabel: AppStrings.AppInformation.privacyImageDescription,
+				accessibilityIdentifier: AccessibilityIdentifiers.AppInformation.privacyImageDescription,
+				height: 230
+			),
+			cells: [
+				.title2(
+					text: AppStrings.AppInformation.privacyTitle,
+					accessibilityIdentifier: AccessibilityIdentifiers.AppInformation.privacyTitle),
+				.html(url: Bundle.main.url(forResource: "privacy-policy", withExtension: "html"))
+			]
+		)
+	])
+
+	static let termsModel = DynamicTableViewModel([
+		.section(
+			header: .image(UIImage(named: "Illu_Appinfo_Nutzungsbedingungen"),
+						   accessibilityLabel: AppStrings.AppInformation.termsImageDescription,
+						   accessibilityIdentifier: AccessibilityIdentifiers.AppInformation.termsImageDescription,
+						   height: 230),
+			cells: [
+				.title2(
+					text: AppStrings.AppInformation.termsTitle,
+					accessibilityIdentifier: AccessibilityIdentifiers.AppInformation.termsTitle),
+				.html(url: Bundle.main.url(forResource: "usage", withExtension: "html"))
+			]
+		)
+	])
+
+}
+
+// return a link if language is German; else return an empty string
+private func germanLinkText() -> String {
+	if Bundle.main.preferredLocalizations.first == "de" {
+		return AppStrings.AppInformation.aboutLink
+	} else {
+		return ""
+	}
```

Comment:
Is there a reason why it's not an optional?

```suggestion
private func germanLinkText() -> String? {
  	if Bundle.main.preferredLocalizations.first == "de" {
  		return AppStrings.AppInformation.aboutLink
  	} else {
  		return nil
  	}
```

Reply:
good idea – changed accordingly